executable-spec/test/Cardano/Ledger/Spec/STS/Chain/Chain/Properties.hs

```diff
@@ -70,52 +70,97 @@ relevantCasesAreCovered
   = QC.withMaxSuccess 300
   $ STS.Gen.forAllTrace @(CHAIN Mock) @() maxTraceLength ()
   $ \traceSample ->
+        (QC..&&.)
+          (STS.Gen.classifyTraceLength maxTraceLength 10 traceSample)
```

Comment:
We are classifying the trace lengths in other tests, so I don't think we want to do it here as well.

Reply:
Yes but we want to classify the length of this specific trace that is used for these tests. No the length in general ...

Reply:
Hmmm, I still think we're classifying a lot of things here. We're using the same function to obtain traces, so the trace lengths should be the same in any case. Maybe we'd like to move the trace length classification to another test (which won't be run from `Main` but from the repl, only if needed). Otherwise we risk putting too much information that'd be hard to grasp (i.e. we might end up staring at a gigantic wall of text wondering what is it telling us).